tests/Feature/Security/SvgLogoUploadXssTest.php

```
@@ -0,0 +1,99 @@
+  </script>
+  <circle cx="50" cy="50" r="40" fill="blue" />
+</svg>
+SVG;
+
+        $file = UploadedFile::fake()->createWithContent('logo.svg', $maliciousSvg);
+
+        // Authenticate as user with "update company" permission
+        $this->actingAs($user);
+
+        // Upload logo - no validation should stop this
+        $response = $this->post('/company/save', [
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$response'. (undefined)

(UnusedLocalVariable)